streams/src/main/java/org/apache/kafka/streams/processor/DefaultPartitionGrouper.java

```diff
@@ -82,7 +82,11 @@ protected int maxNumPartitions(Cluster metadata, Set<String> topics) {
             List<PartitionInfo> partitions = metadata.partitionsForTopic(topic);
 
             if (partitions.isEmpty()) {
-                log.info("Skipping assigning topic {} to tasks since its metadata is not available yet", topic);
+
+                log.warn("Skipping creating tasks for the topic group {} since topic {}'s metadata is not available yet;"
```

Comment:
we do not have the parameter for the topic group in line 89...

Reply:
ack